src/main/webapp/scripts/tests/comment-post-test.js

```diff
@@ -5,7 +5,6 @@ const COMMENT_TEXT = 'sample comment';
 
 QUnit.module('Post a comment via YouTube API', {
     beforeEach: () => {
-        gapi = new CustomGapi();
         sinon.stub(gapi.client, 'request');
```

Comment:
What scope is this gapi from? Is it because the DOM included the other js file that contains it? I'm going to approve this PR but I find it odd that the reader kind of has to guess where this variable is coming from. It would be easier to read to if it was explicitly imported.

Reply:
Done. This gapi was from `<script src="https://apis.google.com/js/api.js"></script>` in tests.html, but I just moved it to the respective test js files for readability.